src/utilities/include/svd.h

```diff
@@ -289,7 +289,7 @@ struct SVDSet {
  *
  * @param A The matrix to decompose.
  */
-SVDSet SVD(glm::mat3 A) {
+inline SVDSet SVD(glm::mat3 A) {
```

Comment:
Should we move this into a .cpp? I'm not completely clear on the tradeoffs.

Reply:
Probably not much pros and cons, more like ideological issue. If this is complicated and used in multiple places, putting it in a `.cpp` will reduce compilation time a bit, but it is not the case here.